mockall/src/lib.rs

```diff
@@ -836,7 +836,8 @@
 //!
 //! Mockall can also mock static methods.  But be careful!  The expectations are
 //! global.  If you want to use a static method in multiple tests, you must
-//! provide your own synchronization.  For ordinary methods, expectations are
+//! provide your own synchronization. See the [`synchronization example`](examples/synchronization.rs)
```

Comment:
This link will be dead, because rustdoc doesn't include files in the examples/ directory.  Instead, it's probably best to link to the file on Github.

Reply:
i just linked to where the file would be once this is merged in. should i link to a specific commit hash or is that fine?

Reply:
I think it's best not to provide a specific hash.  We might update the example later, and it would be easy to forget to update the link.

Reply:
okay, updated to use my previous commit as the reference

Reply:
It looks like you've pinned a specific commit.  Don't you think it's better to point to the latest version?

Reply:
oh, oops. i misread your comment. i thought you wanted me to include a specific commit hash. i can revert that to point to the latest version

Reply:
@asomers does this seem good to go now?